src/main/kotlin/lotto/LottoGame.kt

```diff
@@ -0,0 +1,91 @@
+package lotto
+
+import java.lang.IllegalArgumentException
+import java.lang.NumberFormatException
+import java.lang.StringBuilder
+import java.math.BigDecimal
+import java.util.regex.Pattern
+
+const val COUNT_OF_NUMBERS = 6
+private const val MAX_NUMBER = 45
+private const val PRIZE_COUNT = 3
+private const val MIN_NUMBER = 1
+const val PRICE_OF_LOTTO = 1000
+private const val NUMBER_RANGE = "(4[0-5]|[1-3][0-9]|[1-9])"
+val PRIZE_AMOUNT = listOf(0, 0, 0, 5000, 50000, 1500000, 2000000000)
+
+class LottoGame(amount: String) {
+    private val lottoNumbers: MutableList<List<Int>> = mutableListOf()
+    private var prizes: MutableMap<Int, Lotto> = mutableMapOf()
+    private var prizeAmount: Int = 0
+    var count: Int = 0
+        private set
+    var profitRate: Double = 0.0
+        private set
+
+    private var prizeNumbers: List<Int> = listOf()
+
+    init {
+        checkAmountValidation(amount)
+        PRIZE_AMOUNT.filter { it > 0 }.forEach {
+            prizes[PRIZE_AMOUNT.indexOf(it)] = Lotto(it)
+        }
+    }
+
+    fun execute(prizeNumberString: String): HashMap<Int, Lotto> {
```

Comment:
인터페이스를 타입으로 사용하는 습관을 길러 두면 프로그램이 훨씬 유연해질 거예요. 인터페이스와 다형성에 대해 알아보고 아래의 글도 참고하면 좋을 것 같아요.
https://stackoverflow.com/questions/11683044/return-arraylist-or-list/11683073

```suggestion
    fun execute(prizeNumberString: String): Map<Int, Lotto> {
```